### ISSUE TITLE
Tab label is repeated in list heading. It is also not clear that the number count refers to number for people.

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#31635]
**Steps to repro**
- Go to People

**What we see**
- All the section tabs have a label that is repeated in the list heading below it.

<img width="753" alt="Screen_Shot_2019-03-06_at_11 12 09_AM" src="https://user-images.githubusercontent.com/2124984/54762899-5d767780-4bcb-11e9-83be-0f944cec19fb.png">

**What we suggest**
- Instead of repeating the same label/title, be more obvious about on the list heading. Instead of 'Followers (3)', say 'You have 3 followers'

**Comment by scruffian** [comment]
I think this change need to be made in this file: https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/my-sites/people/followers-list/index.jsx#L217

**Comment by sixhours** [comment]
We chatted about this in Slack, and I've traced the label back to the parent component: 

https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/my-sites/people/main.jsx#L55

But to update the label to use something like "You have X followers", we need to solve for X, which is currently done later down the tree. We could move the follower count logic into the parent component, or use a callback and state to pass it up from the child component. But I'm hoping we already store this value somewhere in redux, so I'm going to look at that first.
